src/Google/Ads/GoogleAds/Lib/V10/GoogleAdsCallLogger.php

```diff
@@ -36,8 +35,16 @@ final class GoogleAdsCallLogger
     private $endpoint;
     private $logMessageFormatter;
     private $context;
-    private static $logLevelList;
-    private static $logLevelNamesToNormalizedValues;
+    private const LOG_LEVELS  = [
```

Comment:
Could you add a comment to explain what these key/values represent?

My understanding is that the keys are all the log levels registered under [RFC 5424](http://tools.ietf.org/html/rfc5424) and the values are the index of severity in increasing order.

If that is the case, what about using an ordered list instead? I think it would makes things easier to understand and implement.